### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by TinaUll**
@zryska  Please take look. Thank you!

### CODE REVIEW

File: content/release-notes/release-notes.md [hunk 1]
```diff
@@ -4,14 +4,26 @@ uid: ReleaseNotes
 
 # Release notes
 
-PI Adapter for OPC UA 1.1
+PI Adapter for OPC UA 1.1.1.4
+Adapter framework 1.4
```
Comment (Xiaoli-OSI) [inline content/release-notes/release-notes.md:8]: Framework version 1.2.0

File: content/release-notes/release-notes.md [hunk 2]
```diff
@@ -4,14 +4,26 @@ uid: ReleaseNotes
 
 # Release notes
 
-PI Adapter for OPC UA 1.1
+PI Adapter for OPC UA 1.1.1.4
```
Comment (Xiaoli-OSI) [inline content/release-notes/release-notes.md:7]: Let's use 1.1.1 without including the last build number.

File: content/release-notes/release-notes.md [hunk 3]
```diff
@@ -4,14 +4,26 @@ uid: ReleaseNotes
...
+Adapter framework 1.4
 
 ## Overview
 
-This represents the initial standalone release for PI Adapter for OPC UA. This product collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, automatic discovery of available data items on a data source, and various Windows and Linux-based operating systems as well as containerization.
+This represents the second release for PI Adapter for OPC UA. This product collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, automatic discovery of available data items on a data source, and various Windows and Linux-based operating systems as well as containerization.
```
Comment (Xiaoli-OSI) [inline content/release-notes/release-notes.md:12]: hot-fix release

**Review by Xiaoli-OSI — APPROVED** [review]

**Review by zryska — APPROVED** [review]

File: content/release-notes/release-notes.md (hunk 2 above)
Comment (zryska) [reply content/release-notes/release-notes.md:7]: Shouldn't this PR be merged into v1.1.1 branch instead of v.1.2?

**Comment by TinaUll** [comment]
Closing without merging because of wrong base branch. Re-opened separate PR.

**Pull request closed without merging**